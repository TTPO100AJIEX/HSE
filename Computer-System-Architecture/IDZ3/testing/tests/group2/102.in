0.25587618437836923