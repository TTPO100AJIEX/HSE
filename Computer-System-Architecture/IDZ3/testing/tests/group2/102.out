0.258753952051999